classes/class-wc-connect-service-settings-store.php

```diff
@@ -124,7 +137,7 @@ public function get_preferred_paper_size() {
 			// According to https://en.wikipedia.org/wiki/Letter_(paper_size) US, Mexico, Canada and Dominican Republic
 			// use "Letter" size, and pretty much all the rest of the world use A4, so those are sensible defaults
 			$base_location = wc_get_base_location();
-			if ( in_array( $base_location[ 'country' ], array( 'US', 'CA', 'MX', 'DO' ) ) ) {
+			if ( in_array( $base_location['country'], array( 'US', 'CA', 'MX', 'DO' ) ) ) {
```

Comment:
I know this line change is unrelated to this PR since it's for a lint fix, but if we're changing it anyway, we could also fix the non-strict in_array comparison here by adding `true` as the third argument. Happy to merge without this fix - up to you.